### ISSUE TITLE
Vanilla build+run: RoutesController required a bean of type DefaultCamelContext that could not be found

### ISSUE THREAD

**Issue by mujina93** [FraunhoferISST/DataspaceConnector#410]
**Describe the bug**
Building and running the connector with docker fails with error (from the logs, when running):

```
2021-07-09T13:28:54,119 [main] ERROR - \n\n***************************\nAPPLICATION FAILED TO START\n***************************\n\nDescription:\n\nParameter 0 of constructor in io.dataspaceconnector.camel.controller.RoutesController required a bean of type 'org.apache.camel.impl.DefaultCamelContext' that could not be found.\n\n\nAction:\n\nConsider defining a bean of type 'org.apache.camel.impl.DefaultCamelContext' in your configuration.\n
```

**To Reproduce**
1. `docker build -t dsc repos/DataspaceConnector`
2. `docker run -p 8080:8080 -d --name dsc-provider dsc:latest`

**Expected behavior**
No errors when trying to do a vanilla build + run.

**Desktop (please complete the following information):**
 - OS: Linux

**Comment by mujina93** [comment]
The problem does not appear when switching to the latest tag

```
git tag --list '*'
git checkout tags/v5.2.1 -b latest
```

**Comment by mujina93** [comment]
Trying v.6.0.0 gives the same error again.

Is there something about Camel to configure in application.properties and/or in conf.json for the connector v.6 to come up properly?

**Comment by brianjahnke** [comment]
Hi,
are you building the connector with dockers buildkit enabled?

**Comment by mujina93** [comment]
Hi. No. Is that a requirement?

**Comment by mujina93** [comment]
I haven't fully understood what's causing the issue, but it disappears when building locally with docker.

The issue is there when building with a remote `DOCKER_HOST`.

**Comment by mujina93** [comment]
I haven't fully understood what's causing the issue, but it disappears when building locally with docker.

The issue is there when building with a remote `DOCKER_HOST`.

**Comment by brianjahnke** [comment]
Hi,
it should be fixed with ce0a355e05702cbfd8074b0c2615fb82ed435413.
